core/client/src/main/java/alluxio/client/file/options/InStreamOptions.java

```diff
@@ -88,6 +91,23 @@ public InStreamOptions setReadType(ReadType readType) {
     return this;
   }
 
+  /**
+   * @return true if incomplete block caching is enabled
+   */
+  public boolean getCacheIncompleteBlock() {
+    return mCacheIncompleteBlock;
+  }
+
+  /**
+   * Enable incomplete block caching.
+   *
+   * @return the updated options object
+   */
+  public InStreamOptions setCacheIncompleteBlock() {
```

Comment:
it seems in the code base, the convention for setters of a boolean field in options takes one param, like
`public LoadMetadataOptions setRecursive(boolean recursive)`